partner_industry_secondary/models/res_partner_industry.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2015 Antiun Ingenieria S.L. - Javier Iniesta
+# Copyright 2016 Tecnativa S.L. - Vicent Cubells
+# Copyright 2016 Tecnativa S.L. - Pedro M. Baeza
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, exceptions, fields, models, _
+
+
+class ResPartnerIndustry(models.Model):
+    _inherit = 'res.partner.industry'
+    _order = "parent_left"
```

Comment:
This should be changed or removed. 
I've got this error when trying to open a Partner in Contacts
```
Error:
Odoo Server Error

Traceback (most recent call last):
  File "/.repo_requirements/odoo/odoo/http.py", line 654, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/.repo_requirements/odoo/odoo/http.py", line 312, in _handle_exception
    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])
  File "/.repo_requirements/odoo/odoo/tools/pycompat.py", line 87, in reraise
    raise value
  File "/.repo_requirements/odoo/odoo/http.py", line 696, in dispatch
    result = self._call_function(**self.params)
  File "/.repo_requirements/odoo/odoo/http.py", line 344, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/.repo_requirements/odoo/odoo/service/model.py", line 97, in wrapper
    return f(dbname, *args, **kwargs)
  File "/.repo_requirements/odoo/odoo/http.py", line 337, in checked_call
    result = self.endpoint(*a, **kw)
  File "/.repo_requirements/odoo/odoo/http.py", line 939, in __call__
    return self.method(*args, **kw)
  File "/.repo_requirements/odoo/odoo/http.py", line 517, in response_wrap
    response = f(*args, **kw)
  File "/home/odoo/OCB-12.0/addons/web/controllers/main.py", line 962, in call_kw
    return self._call_kw(model, method, args, kwargs)
  File "/home/odoo/OCB-12.0/addons/web/controllers/main.py", line 954, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/.repo_requirements/odoo/odoo/api.py", line 749, in call_kw
    return _call_kw_multi(method, model, args, kwargs)
  File "/.repo_requirements/odoo/odoo/api.py", line 736, in _call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/.repo_requirements/odoo/odoo/models.py", line 2766, in read
    self._read_from_database(stored, inherited)
  File "/.repo_requirements/odoo/odoo/models.py", line 2927, in _read_from_database
    field.read(fetched)
  File "/.repo_requirements/odoo/odoo/fields.py", line 2656, in read
    order_by = comodel._generate_order_by(None, wquery)
  File "/.repo_requirements/odoo/odoo/models.py", line 4055, in _generate_order_by
    order_by_elements = self._generate_order_by_inner(self._table, order_spec, query)
  File "/.repo_requirements/odoo/odoo/models.py", line 4022, in _generate_order_by_inner
    raise ValueError(_("Sorting field %s not found on model %s") % (order_field, self._name))
ValueError: Sorting field parent_left not found on model res.partner.industry
```

Reply:
Done!